samples/ftd_configuration/access_rule_with_logging.yml

```diff
@@ -0,0 +1,37 @@
+- hosts: all
+  connection: httpapi
+  tasks:
+
+    - name: Find a file policy blocking all malware
+      ftd_configuration:
+        operation: getFilePolicyList
+        filters:
+          policyType: BLOCK_MALWARE_ALL
+        register_as: file_policies
+
+    - name: Setup a syslog server
+      ftd_configuration:
+        operation: upsertSyslogServer
+        data:
+          name: 10.0.4.5:514
+          useManagementInterface: True
+          protocol: UDP
+          host: 10.0.4.5
+          port: "514"
```

Comment:
do we need quotes here?

Reply:
Yes. Initially, I also put the port value without quotes, but as it's a number, Jinja passed it to Ansible as an integer and our validation complained that Swagger requires the port to be a string, so I had to wrap it in quotes.